src/Host.h

```diff
@@ -125,7 +125,7 @@ class Host : public QObject
     void               unregisterEventHandler(const QString&, TScript * );
     void               raiseEvent( const TEvent & event );
     void               resetProfile();
-    std::tuple<bool, QString, QString, QString, QString> saveProfile();
+    std::tuple<bool, QString, QString, QString> saveProfile(const QString &saveLocation, bool syncModules = false);
```

Comment:
I think that's overengineering it in this case, but should we turn this into a generic pattern, it'd be a good idea!